Automattic/vip-cli
# feat(dev-env): Add support for the most recent Lando

## Description

This PR updates the code so that we can work with the upstream Lando CLI.

Needs Automattic/lando-cli#14; update `package.json` once that PR is merged.

## Steps to Test

1. Check out PR.
2. Run `npm run build`
3. Test dev env-related commands.
